Sources/Controllers/2.0/OAEditFavoriteViewController.mm

```diff
@@ -674,11 +669,13 @@ - (nonnull UITableViewCell *) tableView:(nonnull UITableView *)tableView cellFor
         if ([item[@"isEditable"] boolValue])
```

Comment:
Can be simplified by extracting [item[@"isEditable"] boolValue] to a variable, then setting it to enabled and user interaction. Text color should  be expressed via a ternary comparison